data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -350,6 +351,9 @@ then
 
 	touch mco-bootstrap.done
     record_service_stage_success
+    # Once MCO is running, validate API and API-INT URLs
+    check_api_url
+    check_api_int_url
```

Comment:
Two questions about this:

1. Why wait until after MCO done to run? I have no issue with that, just curious if there is a reason to run after the MCO. If so it would probably be good to add that particular reason to the comment.
2. I think we want this outside the conditional, so just after the `fi` (assuming we want to run after MCO is running). This bootkube script will be rerun, but the logic for starting the MCO will not be executed after it has been successfully started. I believe we want to run the check_api functionality every time we run bootkube.

Reply:
check_api_url and check_api_int_url depend on the API and API_INT urls being available in the Infrastructure CR. IIUC, this CR is populated with these values by the MCO so have to run them after MCO runs successfully in the bootstrap node.
Yes, we would want to run these validations every time bootkube runs. Updating based on #2.